app/components/location-map.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  maps: Ember.inject.service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    Ember.Logger.debug('didInsertElement');
+    this.$('.map-container').append(this.get('maps').getMapElement(this.get('location')));
```

Comment:
I'd be more explicit about naming the service, maps => mapService just so it's clear. Just a thought.